includes/rest-api/class-sensei-rest-api-question-helpers-trait.php

```diff
@@ -69,18 +69,31 @@ private function get_question_schema( string $type ): array {
 	 * @return int|WP_Error Question id on success.
 	 */
 	private function save_question( $question ) {
+		$question_id = $question['id'] ?? null;
+
+		if (
```

Comment:
This is the check that I wanted to highlight in my previous comment.

Reply:
We could do (originally I had it there)... I was more concerned about future uses of `save_question`, but maybe I'll punt on that concern.

In #4032  its return isn't checked and access check happens in the core question controller anyway.